Nomad Docs: Move load balancer content from tutorials to nomad docs (#1523)
hashicorp/web-unified-docs: content/nomad/v1.11.x/content/docs/networking/load-balancer/traefik.mdx

these requests. Traefik's [Consul
Catalog](https://doc.traefik.io/traefik/reference/install-configuration/providers/hashicorp/consul-catalog/)
provides integration and ability to use tags to route traffic. Refer to
Traefik's [Rules & Priorty
documentation](https://doc.traefik.io/traefik/reference/routing-configuration/http/routing/rules-and-priority/)
for details.

Follow these steps to use Traefik as a load balancer:

1. Deploy your application with Traefik-specific tags in the job specification.
1. Deploy Traefik.

This guide shows you how to deploy Traefik as a load balancer for a demo web application.

### Prerequisites

You must configure your Nomad installation to use Consul. Refer to the
[Consul integration](/nomad/docs/networking/consul) content for more
information.

## Deploy the demo web application

Create a job specification for a demo web application and name the file
`webapp.nomad.hcl`. This job specification configures the following:

- Three instances of the demo web application. 
- Routing to the web app with service tags. Even though the application
  listens on `/`, it is possible to define `/myapp` as the route because of

>   Traefik's [`Path` option](https://docs.traefik.io/routing/routers/#rule).

Reviewer: ```suggestion
  [Traefik's `Path` option](https://docs.traefik.io/routing/routers/#rule).
```
IMO if it's linking to external docs, the product name should be part of the link text

      address = "127.0.0.1:8500"
      scheme  = "http"
EOF

        destination = "local/traefik.toml"
      }

      resources {
        cpu    = 100
        memory = 128
      }
    }
  }
}
```

</CodeBlockConfig>

Although the job specification contains an inline template for Traefik
configuration, you could alternatively use the `task.template` stanza in
conjunction with the `task.artifact` stanza to download an input template from a
remote source such as an S3 bucket. Refer to the [`template` examples](/nomad/docs/job-specification/template#examples) for details.

Deploy the Traefik job with the `nomad run` command.

```shell-session
$ nomad run traefik.nomad.hcl
==> Monitoring evaluation "e22ce276"
    Evaluation triggered by job "traefik"
    Evaluation within deployment: "c6466497"
    Allocation "695c5632" created: node "a62fa99d", group "traefik"
    Evaluation status changed: "pending" -> "complete"
==> Evaluation "e22ce276" finished with status "complete"
```

### Check the Traefik dashboard

Visit the Traefik dashboard at`http://<your-traefik-address>:8081` to verify
your settings and for basic monitoring.

## Make a request to the load balancer

> If you access the Traefik load balancer, you should be able to see a response

Reviewer: ```suggestion
If you access the Traefik load balancer, you should receive a response
```
